openwisp_firmware_upgrader/api/views.py

```diff
@@ -33,8 +33,9 @@ class OrgAPIMixin(ProtectedAPIMixin):
     def get_queryset(self):
         queryset = self.queryset.all().order_by('-pk')
         if not self.request.user.is_superuser:
-            user_org = self.request.user.openwisp_users_organization.get()
-            organization_filter = {self.organization_field: user_org}
+            user_orgs = self.request.user.openwisp_users_organization.all()
```

Comment:
I took a note to remind us to cache this: https://github.com/openwisp/openwisp-firmware-upgrader/issues/59